app/views/purchases/index.html.erb

```diff
@@ -1,34 +1,75 @@
-<%= render 'header' %>
+<h1 class="title">Purchases</h1>
 
-<div class='purchase-new'>
-  <%= render 'form' %>
+<div class="card">
+  <div class="card-content">
+    <%= render 'form' %>
+  </div>
 </div>
 
+<br>
+
 <div class='purchase-index'>
-  <table>
-    <thead>
-      <tr>
-        <th>Date</th>
-        <th>Description</th>
-        <th>Amount</th>
-        <th>Payee</th>
-        <th>Category</th>
-        <th>Purchaser</th>
-      </tr>
-    </thead>
-    <tbody>
-    <% if @purchases %>
-      <% @purchases.order(:date).reverse_order.each do |purchase| %>
+  <div>
+    <table class="table is-bordered is-striped is-hidden-mobile">
```

Comment:
I didn't know about the Bulma `is-hidden-mobile` class (and related ones) - nice!